bolt/lib/Profile/StaleProfileMatching.cpp

```diff
@@ -180,6 +186,17 @@ struct BlendedBlockHash {
   uint8_t SuccHash{0};
 };
 
+/// A data object containing function matching information.
+struct FunctionMatchingData {
+public:
+  /// The number of blocks matched exactly.
+  uint64_t MatchedExactBlocks{0};
+  /// The number of blocks matched loosely.
+  uint64_t MatchedLooseBlocks{0};
+  /// The number of execution counts matched.
+  uint64_t MatchedExecCounts{0};
```

Comment:
What's the intended use for these two? There is no usage anywhere right now, and usually we don't keep unused stuff around.

Reply:
Left this for possible extensibility of our definition for the threshold, but since these aren't being used at the moment, I'll remove them